agenda.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>WiDS - Agenda</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <header class="container">
+    <!-- Start Navbar Section -->
+    <nav class="navbar navbar-expand-md navbar-light">
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse"
+        aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarCollapse">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item">
+            <a class="nav-link" href="landing.html">Home</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link text-success active" href="agenda.html">Schedule</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="speakers.html">Speakers</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="venue.html">Venue</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Contact</a>
+          </li>
+        </ul>
+        <div class="mt-2 mt-md-0">
+          <button class="btn btn-success ">Get a ticket!</button>
+        </div>
+      </div>
+    </nav>
+    <!-- End Navbar Section -->
+  </header>
+  <!-- Start Agenda Section -->
+  <section class="container">
+    <h4 class="text-success text-center"><strong>WDiS POZNAŃ SATELITE EVENT</strong></h4>
+    <h5 class="text-center">13.04.2019, Poznań University od Technology</h5>
+    <h3 class="text-center font-weight-bold">Agenda</h3>
```

Comment:
header order 
h2 > h3 > h4 
better idea - use h2 or h3 and the rest style p with your own class css